pkg/webhook/channel.go

```diff
@@ -0,0 +1,92 @@
+package webhook
```

Comment:
can we move this into another package pkg/tools/channel.go ?

Reply:
I think in golang, its an antipattern to put funcs in packages named "tools" or "utils". 
["Name your packages after what they provide, not what they contain."](https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common)
However, having everything in package webhook is not good either and something I want to improve. I will move the channel stuff into its package. Thanks @pperiyasamy

Reply:
good info! I will follow this convention ;)